Index: src/main/java/com/example/florarie_proiect/model/Bouquet.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.florarie_proiect.model;public class Bouquet {\r\n}\r\n
===================================================================
diff --git a/src/main/java/com/example/florarie_proiect/model/Bouquet.java b/src/main/java/com/example/florarie_proiect/model/Bouquet.java
--- a/src/main/java/com/example/florarie_proiect/model/Bouquet.java	
+++ b/src/main/java/com/example/florarie_proiect/model/Bouquet.java	
@@ -1,2 +1,39 @@
-package com.example.florarie_proiect.model;public class Bouquet {
+package com.example.florarie_proiect.model;
+
+public class Bouquet {
+    private String name;
+    private int price;
+    private int quantity;
+
+    public Bouquet(String name, int quantity, int price) {
+        this.name = name;
+        this.quantity = quantity;
+        this.price=price;
+    }
+
+    // getters and setters
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = price;
+    }
+
+    public int getQuantity(){
+        return quantity;
+    }
+
+    public void setQuantity(int quantity){
+        this.quantity=quantity;
+    }
 }
+
Index: src/main/resources/com/example/florarie_proiect/modify.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n\r\n<?import java.lang.*?>\r\n<?import java.util.*?>\r\n<?import javafx.scene.*?>\r\n<?import javafx.scene.control.*?>\r\n<?import javafx.scene.layout.*?>\r\n\r\n<AnchorPane xmlns=\"http://javafx.com/javafx\"\r\n            xmlns:fx=\"http://javafx.com/fxml\"\r\n            fx:controller=\"com.example.florarie_proiect.Modify\"\r\n            prefHeight=\"400.0\" prefWidth=\"600.0\" >\r\n\r\n</AnchorPane>\r\n
===================================================================
diff --git a/src/main/resources/com/example/florarie_proiect/modify.fxml b/src/main/resources/com/example/florarie_proiect/modify.fxml
--- a/src/main/resources/com/example/florarie_proiect/modify.fxml	
+++ b/src/main/resources/com/example/florarie_proiect/modify.fxml	
@@ -1,14 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import java.lang.*?>
-<?import java.util.*?>
-<?import javafx.scene.*?>
 <?import javafx.scene.control.*?>
 <?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
 
-<AnchorPane xmlns="http://javafx.com/javafx"
-            xmlns:fx="http://javafx.com/fxml"
-            fx:controller="com.example.florarie_proiect.Modify"
-            prefHeight="400.0" prefWidth="600.0" >
-
+<AnchorPane prefHeight="368.0" prefWidth="512.0" xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.florarie_proiect.Modify">
+   <children>
+      <Label layoutX="221.0" layoutY="45.0" prefHeight="27.0" prefWidth="150.0" text="Modify bouquet">
+         <font>
+            <Font size="18.0" />
+         </font></Label>
+      <Label layoutX="67.0" layoutY="192.0" text="New Price" />
+      <Label layoutX="67.0" layoutY="250.0" text="New Quantity" />
+      <Label layoutX="67.0" layoutY="145.0" prefHeight="18.0" prefWidth="86.0" text="New Name" />
+      <Label layoutX="67.0" layoutY="102.0" text="Choose Bouquet" />
+      <ChoiceBox layoutX="215.0" layoutY="98.0" prefWidth="150.0" />
+      <TextField layoutX="216.0" layoutY="142.0" />
+      <TextField layoutX="216.0" layoutY="197.0" />
+      <TextField layoutX="216.0" layoutY="255.0" />
+      <Button layoutX="230.0" layoutY="303.0" mnemonicParsing="false" text="Save Changes" />
+   </children>
 </AnchorPane>
